Modules/Scripted/VectorToScalarVolume/VectorToScalarVolume.py

```diff
@@ -82,18 +79,27 @@ def __init__(self, parent):
 #
 # VectorToScalarVolumeParameterNode
 #
-_LUMINANCE = 'LUMINANCE'
-_AVERAGE = 'AVERAGE'
-_SINGLE_COMPONENT = 'SINGLE_COMPONENT'
 
-_CONVERSION_METHODS = (_LUMINANCE, _AVERAGE, _SINGLE_COMPONENT)
+class ConversionMethods(enum.Enum):
+    LUMINANCE = (
+        "Luminance",
+        "(RGB,RGBA) Luminance from first three components: 0.30*R + 0.59*G + 0.11*B + 0.0*A)",
+    )
+    AVERAGE = (
+        "Average",
+        "Average all the components.",
+    )
+    SINGLE_COMPONENT = (
+        "Single Component Extraction",
+        "Extract single component",
+    )
 
 
 @parameterNodeWrapper
 class VectorToScalarVolumeParameterNode:
     InputVolume: slicer.vtkMRMLVectorVolumeNode
     OutputVolume: slicer.vtkMRMLScalarVolumeNode
-    ConversionMethod: Annotated[str, Choice(_CONVERSION_METHODS), Default(_LUMINANCE)]
+    ConversionMethod: ConversionMethods
```

Comment:
I think this will not break loading an old MRML scene. Please confirm.

Reply:
Because the enum names `LUMINANCE`, `SINGLE_COMPONENT` are the same as the old string values `_LUMINANCE = "LUMINANCE"`, `_SINGLE_COMPONENT = "SINGLE_COMPONENT"` the parameter node contents are unchanged.